latex/epoch.tex

```diff
@@ -315,25 +299,13 @@ \subsection{Stake Distribution Calculation}
       & \where \\
       & ~~~~ (\var{stkeys},~\var{rewards},~\var{delegations},~\var{ptrs}) = \var{dstate} \\
       & ~~~~ (\var{stpools},~\wcard,~\wcard,~\wcard) = \var{pstate} \\
-      & ~~~~ \var{outs} = \fun{consolidate}~{utxo} \\
-      & ~~~~ \var{stake} = (\fun{baseStake}~{outs})
-                             \unionoverridePlus (\fun{ptrStake}~{outs}~{ptrs})
-                             \unionoverridePlus (\fun{rewardStake}~{rewards}) \\
+      & ~~~~ \var{stake} = \fun{aggregate_{+}}~
+        \left(\fun{utxoRel}~\var{utxo}\right)\circ\left(\fun{hkRel}~\var{ptrs}\right) \\
```

Comment:
Did we introduce function composition? I guess we take the definition in which the first function being applied is to the right of the composition symbol ...

Reply:
ah yes, thank you, I forgot to define composition of relations.